src/adapters/lokijs/test.js

```diff
@@ -9,17 +9,18 @@ import DatabaseAdapterCompat from '../compat'
 describe('LokiJSAdapter (Synchronous / Memory persistence)', () => {
   commonTests().forEach((testCase) => {
     const [name, test] = testCase
-
+    const dbName = `test${Math.random()}`
```

Comment:
why? it's not wrong here per se, since this is memory-backed, but potentially error-prone since you don't want that in tests with actual persistent storage (or _shared_ memory storage as in sqlite)

Reply:
fixed: https://github.com/Nozbe/WatermelonDB/pull/1227/commits/61ecb6b94dfd2c28131652707bb4f86ee0ab70e6